### ISSUE TITLE
Allow shutdown from backend of dotCMS

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#19319]
In containerized environments and in hosted environments where a failed dotCMS service restarts itself, some customers want a way to terminate an existing dotCMS instance so that it can be rebuilt by the environment/orchestrator.  This can be done via an api call or from the console.

**Comment by dsilvam** [comment]
PR: https://github.com/dotCMS/core/pull/19339

**Comment by freddyucv** [comment]
PR: https://github.com/dotCMS/core/pull/19492

**Comment by dsilvam** [comment]
**Passed Internal QA**: System shutdown after hitting the new endpoint.

Request:
```
$ curl -XGET http://localhost:8080/api/v1/maintenance/_scurl -XDELETE http://localhost:8080/api/v1/maintenance/_shutdown -u admin@dotcms.com:admin
```
Response:
```
{"entity":"Shutdown","errors":[],"i18nMessagesMap":{},"messages":[],"permissions":[]}
```

**Comment by bryanboza** [comment]
We need to add some postman tests here...

**Comment by dsilvam** [comment]
@bryanboza this is not possible since it shuts down the server.

**Comment by bryanboza** [comment]
@dsilvam we already talk about this, and we can generate the test regarding permissions, response codes and all the stuff around the endpoint, just leave the success case.

**Comment by dsilvam** [comment]
@bryanboza postman test included.

**Comment by bryanboza** [comment]
Fixed, tested on release-20.11 // Postgres // FF
